examples/blog/layouts/partials/meta.html

```diff
@@ -1,6 +1,5 @@
 
     <meta charset="utf-8">
-    <meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <meta name="description" content="{{ .Description }}">
-    <meta name="author" content="A Hugo User"> <!-- This should be modified to be your name, if you want to include this information -->
\ No newline at end of file
+    <meta name="author" content="A Hugo User"> <!-- This should be modified to be your name, if you want to include this information -->
```

Comment:
We should update this to say Gotham instead.

Reply:
Why?

Reply:
Our users would see "A Hugo User" there. Your call, we should review this outside of this PR, as this has been like that before this update.

Reply:
I opened this back up. I question if users would even see this. I thought it was more of like a test, though it's not named that. Now I'm not sure.